packages/core/src/stepFunctions/workflowStudio/handleMessage.ts

```diff
@@ -0,0 +1,43 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Command, Message, MessageType, SaveFileRequestMessage, WebviewContext } from './types'
+import { saveFileMessageHandler, autoSaveFileMessageHandler } from './handlers/saveFileMessageHandler'
+import { initMessageHandler } from './handlers/initMessageHandler'
+import { loadStageMessageHandler } from './handlers/loadStageMessageHandler'
+import { openFeedbackMessageHandler } from './handlers/openFeedbackMessageHandler'
```

Comment:
do we need a file for each handler? Can we just put them all in this file?

Reply:
Putting all of them in the same file is definitely and option, I just thought it was the best practice to keep them in separate files for more clarity (looking at the examples of how it's implemented for Application Composer and Threat Composer integrations in the similar `handleMessage.ts` files).

However, I'm open to consolidating all handlers into this file if you think it’s a better approach. My main concern is ensuring consistency, as for other integrations the handlers are separated

Reply:
Generally we prefer less files so that its easier to track and understand code. It also helps reduce the chances of circular dependencies.
App composer's approach here seems to be a miss in our codebase- we should do the same refactor there if we touch that code again.